soul-kk/.env

```
@@ -0,0 +1,4 @@
+VITE_API_URL = 'https://ark.cn-beijing.volces.com/api/v3/chat/completions'
+VITE_API_KEY = '7508957e-f3c3-4364-8d7f-d92a233fe61b'
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)